pkg/JuliaInterface/init.g

```diff
@@ -4,22 +4,44 @@
 # Reading the declaration part of the package.
 #
 
-##
-##  If the value is <K>true</K> then the file
-##  <F>~/.julia/config/startup.jl</F>
-##  gets imported into &Julia; after startup.
-##  If the value is a nonempty string that is the name of a directory
-##  then the <F>startup.jl</F> file in this directory gets imported into
-##  &Julia;.
-##  Otherwise no <F>startup.jl</F> file will be imported automatically,
-##  since apparently <C>libjulia</C> does not import such a file.
+#! @BeginChunk IncludeJuliaStartupFile
+#!  <Subsection Label="subsect:IncludeJuliaStartupFile">
+#!  <Heading>User preference <C>IncludeJuliaStartupFile</C></Heading>
+#!  <Index Key="IncludeJuliaStartupFile"><C>IncludeJuliaStartupFile</C></Index>
+#!
+#!  When one starts an interactive &Julia; session,
+#!  the &Julia; startup file <F>~/.julia/config/startup.jl</F> gets
+#!  included automatically by default,
+#!  see the section <Q>The Julia REPL</Q> in the &Julia; documentation.
```

Comment:
Perhaps this could be turned into an actual link to https://docs.julialang.org/en/v1/stdlib/REPL/?

Reply:
Yes.